spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua

```diff
@@ -3,7 +3,7 @@ local helpers      = require "spec.helpers"
 local ssl_fixtures = require "spec.fixtures.ssl"
 
 
-local POLL_INTERVAL = 0.3
```

Comment:
this POLL_INTERVAL also controls `db_update_frequency`, meaning that if we raise this number, then the update frequency gets _slower_.

If you only want to add more time to the propagation delay, I'd recommend doing it in line 65 instead. 

But the way to to make this test file magic-sleep-number-independent would be to replace those `wait_for_propagation()` delays with using `helpers.wait_until()` instead (essentially remove the `wait_for_propagation()` call and wrap the API call below it with `helpers.wait_until(function() ... end)`
( [03-plugins/25-oauth2/04-invalidations_spec.lua](https://github.com/Kong/kong/blob/master/spec/03-plugins/25-oauth2/04-invalidations_spec.lua#L148-L155) is a good example)